src/executables/util_cs.c

```diff
@@ -3218,6 +3217,20 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   check_replica_info = (HA_GET_MODE () == HA_MODE_REPLICA);
   pageid = utility_get_option_bigint_value (arg_map, APPLYINFO_PAGE_S);
+  while (arg->argv[argv_index] != NULL)
```

Comment:
I think `utility_get_option_index` can be used here.
Using above function seems to be easier to understand to read the code.

Reply:
What happens when the user uses "--pageid"?

Reply:
> I think `utility_get_option_index` can be used here. Using above function seems to be easier to understand to read the code.

It cannot use here.
`arg` includes `arg->arg_map`. So I cannot get the information about the command what I type in the `utility_get_option_index`

Reply:
> 

I may guess you mean that 
`cubrid applyinfo -L [directory] -p` right? and then is there anything to type the page id?